vehicle-control/simulation/Assets/Standard Assets/Vehicles/Car/Prefabs/Car.prefab

```diff
@@ -535,6 +535,24 @@ GameObject:
   - 4: {fileID: 466396}
   - 114: {fileID: 11458312}
   - 120: {fileID: 12050378}
+  - 114: {fileID: 11465580}
+  m_Layer: 0
+  m_Name: DepthCollectionSensor
```

Comment:
I think this change belongs in the other PR (https://github.com/numenta/nupic.research/pull/191/files)